### PR TITLE
Update the build to SBT 1.x

### PR DESCRIPTION
**Pull request by filosganga**
Update the build and plugins to SBT 1.x
It also recreate all the file headers

### CODE REVIEW

File: avro/src/main/java/com/ovoenergy/kafka/serialization/avro/JerseyClientRegisterHack.java [hunk 1]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2017 OVO Energy
```
Comment (jCalamari) [inline avro/src/main/java/com/ovoenergy/kafka/serialization/avro/JerseyClientRegisterHack.java]: 2017-2018 maybe?

File: avro4s/src/main/scala/com/ovoenergy/kafka/serialization/avro4s/Avro4sSerialization.scala [hunk 2]
```diff
@@ -265,10 +281,8 @@ private[avro4s] trait Avro4sSerialization {
                                                              isKey: Boolean): KafkaDeserializer[T] =
     avroJsonSchemaIdDeserializer(SchemaRegistryClientSettings(schemaRegistryEndpoint), isKey)
 
-  def avroJsonSchemaIdDeserializer[T: FromRecord: SchemaFor](
-    schemaRegistryClientSettings: SchemaRegistryClientSettings,
-    isKey: Boolean
-  ): KafkaDeserializer[T] = {
+  def avroJsonSchemaIdDeserializer[T: FromRecord: SchemaFor](schemaRegistryClientSettings: SchemaRegistryClientSettings,
```
Comment (jCalamari) [inline avro4s/src/main/scala/com/ovoenergy/kafka/serialization/avro4s/Avro4sSerialization.scala]: Is it scala-style formatting? I preferred the previous one.

File: avro/src/main/java/com/ovoenergy/kafka/serialization/avro/JerseyClientRegisterHack.java (hunk 1 above)
Comment (filosganga) [reply avro/src/main/java/com/ovoenergy/kafka/serialization/avro/JerseyClientRegisterHack.java]: yup

File: avro4s/src/main/scala/com/ovoenergy/kafka/serialization/avro4s/Avro4sSerialization.scala (hunk 2 above)
Comment (filosganga) [reply avro4s/src/main/scala/com/ovoenergy/kafka/serialization/avro4s/Avro4sSerialization.scala]: I prefer the old one as well, Have to play around with scalafmt to make it work

File: avro/src/main/java/com/ovoenergy/kafka/serialization/avro/JerseyClientRegisterHack.java (hunk 1 above)
Comment (filosganga) [reply avro/src/main/java/com/ovoenergy/kafka/serialization/avro/JerseyClientRegisterHack.java]: the sbt-header plugin does not support it.

**Pull request merged**